tests.edn

```diff
@@ -0,0 +1,5 @@
+#kaocha/v1
+{:plugins [:kaocha.plugin/profiling
+           :kaocha.plugin/print-invocations
+           :kaocha.plugin/cloverage
+           #_:kaocha.plugin.alpha/spec-test-check]}
```

Comment:
Let's either enable or remove this.

Reply:
I would like to start using that but it caused issues in specific side-effecting functions.

Reply:
> By default, this plugin also adds :no-gen to :kaocha.filter/skip-meta. You might want to decorate an fdef-ed function with ^:no-gen if there is either no good generator for one or more of its arguments or if the function is side-effectful.
https://cljdoc.org/d/lambdaisland/kaocha/0.0-565/doc/automatic-spec-test-check-generation